$NetBSD$

Set the host environment variables in the chirpstack configuration file to
localhost since the post install script which do it for rpm package generation
is not run.

--- chirpstack/configuration/chirpstack.toml.orig	2023-09-11 15:58:25.000000000 +0000
+++ chirpstack/configuration/chirpstack.toml
@@ -19,7 +19,7 @@
   #  * require - Always SSL (skip verification)
   #  * verify-ca - Always SSL (verify that the certificate presented by the server was signed by a trusted CA)
   #  * verify-full - Always SSL (verify that the certification presented by the server was signed by a trusted CA and the server host name matches the one in the certificate)
-  dsn="postgres://chirpstack:chirpstack@$POSTGRESQL_HOST/chirpstack?sslmode=disable"
+  dsn="postgres://chirpstack:chirpstack@localhost/chirpstack?sslmode=disable"
 
   # Max open connections.
   #
@@ -41,7 +41,7 @@
   #
   # Set multiple addresses when connecting to a cluster.
   servers=[
-    "redis://$REDIS_HOST/",
+    "redis://localhost/",
   ]
 
   # Redis Cluster.
@@ -99,5 +99,5 @@
   enabled=["mqtt"]
 
   [integration.mqtt]
-    server="tcp://$MQTT_BROKER_HOST:1883/"
+    server="tcp://localhost:1883/"
     json=true
